app/files/models/file.py

```diff
@@ -0,0 +1,72 @@
+from django.db import models
+from django.db.models import PROTECT
+
+from app.common.enums import AdminGroup, Groups
+from app.common.permissions import BasePermissionModel, check_has_access
+from app.files.models.user_gallery import UserGallery
+from app.util.models import BaseModel
+
+
+class File(BaseModel, BasePermissionModel):
+    read_access = AdminGroup.admin()
+    write_access = AdminGroup.admin()
+
+    title = models.CharField(max_length=80)
+
+    url = models.URLField()
+    description = models.TextField(blank=True)
+    gallery = models.ForeignKey(
+        UserGallery, on_delete=PROTECT, related_name="files", blank=False
+    )
```

Comment:
Url her burde fjernes. Heller lag en ny klasse som er helt lik som OptionalImage, bare OptionalFile, med file som key istedenfor image. Da bruker du denne og så fikser den oppdatering av fil for deg. Alt dere skal ha er url til filen ikke selve  filen.